solc/wrapper.py

```diff
@@ -80,6 +81,14 @@ def solc_wrapper(solc_binary=SOLC_BINARY,
     if assemble:
         command.append('--assemble')
 
+    if remappings:
+        assert type(remappings) in (tuple, list), "imports must be in Python list or tuple"
```

Comment:
I have avoided assertions in production code because they can actually be disabled at the interpreter level.  Can you change these to raise `ValueError` or `TypeError` exceptions depending on what you think is appropriate.